spring-cloud-gcp-autoconfigure/src/main/java/com/google/cloud/spring/autoconfigure/firestore/GcpFirestoreProperties.java

```diff
@@ -82,7 +92,8 @@ public void setEmulator(FirestoreEmulatorProperties emulator) {
   }
 
   public String getFirestoreRootPath(GcpProjectIdProvider projectIdProvider) {
-    return String.format(ROOT_PATH_FORMAT, this.getResolvedProjectId(projectIdProvider));
+    return String.format(ROOT_PATH_FORMAT, this.getResolvedProjectId(projectIdProvider),
+        this.getDatabaseId() == null ? "(default)" : this.getDatabaseId());
```

Comment:
Nice to have: This can be extracted to a method `getResolvedDatabaseId()`, similar to `getResolvedProjectId()`.